installer/scripts/service_control

```diff
@@ -69,72 +187,142 @@ verify_privileges()
     fi
 }
 
-is_omsagent_running()
+this_omsagent_running()
 {
-    verify_privileges
+    local omsagent_pid=`cat $PIDFILE 2>/dev/null`
+    local ps_state=
+    WS_STATUS=0
+    if [ -f $PIDFILE ]; then
+        ps_state=`ps --no-header -o state -p $omsagent_pid`
+        if [ -z "$ps_state" ]; then
+            WS_STATUS=1 # No.
+        else
+            case "$ps_state" in
+            D)  echo "Uninterruptable Sleep State Seen in omsagent process.";;
```

Comment:
Yes, I find constructing short cases with simple columnar alignments optimizes maintainability.  Looking at this now, but it may require a risky addition to get that.

Reply:
That one was different than I thought.  Got the alignment as you suggested.